src/client/src/components/pages/login/index.jsx

```diff
@@ -10,9 +10,12 @@ class LoginForm extends Component {
     username: "",
     password: ""
   };
-  componentWillMount() {
-    this.props.history.push("/login");
-  }
+  componentWillMount = () => {
+    const user = JSON.parse(localStorage.getItem("user"));
+    if (!user || !user.logging) {
+      this.props.history.push("/login");
```

Comment:
@knanahassouna1 
I don't totally understand the reason for the redirect - if the user is viewing this component, aren't they already at /login?